src/app/pages/wallets/management/pages-wallets-management.component.scss

```diff
@@ -10,6 +10,12 @@
   @media only screen and (min-width: 900px) {
     grid-template-columns: 1fr 1fr;
   }
+
+  app-my-wallet:last-of-type {
```

Comment:
I think that instead adding margin to this element you can go with padding for the container that holds this element. It will be cleaner for example if we will add another element, layout will change or we will rename this element.